README.md

```diff
@@ -115,11 +115,8 @@ nmstatectl edit eth3
 
 ## Contact
 
-*Nmstate* uses the [NetworkManager mailing
-list](https://mail.gnome.org/mailman/listinfo/networkmanager-list)
-([Archives](https://mail.gnome.org/archives/networkmanager-list/)) for
-discussions. Emails about nmstate should be tagged with `[nmstate]` in the
-subject header to ease filtering.
+*Nmstate* uses the [nmstate-devel@lists.fedorahosted.org][mailing_list] for
+discussions.
```

Comment:
Can we add below this the instruction on how to register (at least the simplest one)?

```
To subscribe you can send an email with 'subscribe' in the subject to nmstate-devel-join@lists.fedorahosted.org
or visit the mailing list page.
```